commit 33592352b6b346a5fcbf09935da18f994a6babf9 (refs/changes/45/545/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-28T17:43:14-07:00 (3 years ago)
    
    ADMINUI-2336 Update sdcnode version for OpenSSL 1.0.2j/1.0.1u vulnerability fixes
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/Makefile b/Makefile
index 8fff66be..f8039bf7 100644
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ JSSTYLE_FLAGS    = -o "indent=2,doxygen,unparenthesized-return=0,line-length=120
 REPO_MODULES	 =
 SMF_MANIFESTS_IN = smf/manifests/adminui.xml.in
 
-NODE_PREBUILT_VERSION=v0.10.45
+NODE_PREBUILT_VERSION=v0.10.47
 NODE_PREBUILT_TAG=zone
 NODE_PREBUILT_IMAGE=fd2cc906-8938-11e3-beab-4359c665ac99
 
